editor/editor_help.cpp

```diff
@@ -1784,12 +1788,10 @@ EditorHelp::EditorHelp() {
 	//class_list->set_selection_enabled(true);
 
 	{
-		background_panel = memnew(Panel);
-		background_panel->set_v_size_flags(SIZE_EXPAND_FILL);
-		vbc->add_child(background_panel);
 		class_desc = memnew(RichTextLabel);
-		background_panel->add_child(class_desc);
```

Comment:
I think I also was thinking about removing the background panel. But it has it's need when you set the code theme background color to transparent to use a color which fits to all the themes

Reply:
i dont see the problem.

Reply:
Maybe you changed that, but the background color of rtl was overridden by the code theme background color. Which made sense to have the help + the code editor + help bit in the same style. (same background and font colors) but if you wanted to have a color which fits to the theme dark_color you had the option to just make it transparent. So the panel underneath was visible. If the panel is removed making it transparent looks wired...

Reply:
@toger5 yes, now that variation is directly applied to the EditorHelp, and i guess the Script editor, so if you set alpha values to the script bg color you get what you want, i dont see the need for a panel behind.